my-app/src/pages/newPostPage/AddPostPage.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {useCallback, useContext, useEffect, useRef, useState} from 'react';
+import {Button, Paper, TextField} from "@mui/material";
+import styled from "styled-components";
+import {NewPostContext} from "./NewPostContext";
+import "easymde/dist/easymde.min.css";
+import SimpleMdeReact from "react-simplemde-editor";
+
+
+export const AddPostPage = () => {
+    const logic = useContext(NewPostContext)
+    const inputFileRef = useRef<HTMLInputElement | null>(null)
+
+    return (
+        <Wrapper style={{padding: 30}}>

```

Comment:
also don't use inline styles, keep one style of writing your css. If you chose styled components stick to it